packages/@okta/vuepress-site/docs/reference/api/iam-roles/index.md

```diff
@@ -0,0 +1,3027 @@
+            }
+        }
+    }
+]
+```
+
+### Assign a role
+
+#### Assign a role to a user
+
+
+<ApiOperation method="post" url="/api/v1/users/${userId}/roles" />
+
+Assigns a Role to a User
+
+##### Request parameters
+
+
+| Parameter   | Description              | Param Type   | DataType                    | Required |
+| :---------- | :----------------------- | :----------- | :-------------------------- | :------- |
+| type        | type of Role to assign   | Body         | [Role Type](#role-types)    | TRUE     |
+| userId      | `id` of a User           | URL          | String                      | TRUE     |
+
+##### Response parameters
+
+
+Assigned [Role](#role-object)
+
+##### Request example
+
+
+```bash
+curl -v -X POST \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+-d '{
+      "type": "SUPER_ADMIN"
+}' "https://${yourOktaDomain}/api/v1/users/00u6fud33CXDPBXULRNG/roles"
+```
+
+###### Response example
+
+
+```json
+{
+  "id": "ra1b8anIk7rx7em7L0g4",
+  "label": "Super Organization Administrator",
+  "type": "SUPER_ADMIN",
+  "status": "ACTIVE",
+  "created": "2015-09-06T15:28:47.000Z",
+  "lastUpdated": "2015-09-06T15:28:47.000Z"
+}
+```
+
+#### Assign a Role to a Group
+
+
+<ApiOperation method="post" url="/api/v1/groups/${groupId}/roles" />
+
+Assigns a Role to a Group
+
+##### Request parameters
+
+
+| Parameter       | Description                 | Param Type   | DataType                    | Required |
+| :-------------- | :-------------------------- | :----------- | :-------------------------- | :------- |
+| groupId         | `id` of a Group             | URL          | String                      | TRUE     |
+| type            | type of Role to assign      | Body         | [Role Type](#role-types)    | TRUE     |
+
+##### Response parameters
+
+
+Assigned [Role](#role-object)
+
+##### Request example
+
+
+```bash
+curl -v -X POST \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+-d '{
+      "type": "ORG_ADMIN"
+}' "https://${yourOktaDomain}/api/v1/groups/00gsr2IepS8YhHRFf0g3/roles"
+```
+
+###### Response example
+
+
+```json
+{
+    "id": "grasraHPx7i79ajaJ0g3",
+    "label": "Organization Administrator",
+    "type": "ORG_ADMIN",
+    "status": "ACTIVE",
+    "created": "2019-02-27T14:56:55.000Z",
+    "lastUpdated": "2019-02-27T14:56:55.000Z",
+    "assignmentType": "GROUP",
+    "_links": {
+        "assignee": {
+            "href": "http://${yourOktaDomain}/api/v1/groups/00gsr2IepS8YhHRFf0g3"
+        }
+    }
+}
+```
+
+#### Assign a custom role to a user or group
+<ApiLifecycle access="beta" />
+
+The recommended way to assign a Custom Role is by using one of the [Custom Role assignment operations](#custom-role-assignment-operations). However, it is also possible to use the following to assign them too:
```

Comment:
However, it is also possible to use the following to assign them too: > However, you can also assign a Custom Role using the following: